scripts/release/test/util/test_package.sh

```diff
@@ -24,41 +23,11 @@ OS_LIST=(
     ubuntu:18.04
 )
 
-BRANCH=
-CHANNEL=stable
-HASH=
-RELEASE=
 FAILED=()
 
-while [ "$1" != "" ]; do
-    case "$1" in
-        -b)
-            shift
-            BRANCH="$1"
-            ;;
-        -c)
-            shift
-            CHANNEL="$1"
-            ;;
-        -h)
-            shift
-            HASH="$1"
-            ;;
-        -r)
-            shift
-            RELEASE="$1"
-            ;;
-        *)
-            echo "$RED_FG[$0]$END_FG_COLOR Unknown option $1"
-            exit 1
-            ;;
-    esac
-    shift
-done
-
-if [ -z "$BRANCH" ] || [ -z "$HASH" ] || [ -z "$RELEASE" ]
+if [ -z "$BRANCH" ] || [ -z "$COMMIT_HASH" ] || [ -z "$FULLVERSION" ]
 then
-    echo "$YELLOW_FG[Usage]$END_FG_COLOR $0 -b BRANCH -c CHANNEL -h HASH -r RELEASE"
+    echo "[ERROR] $0 was not provided with BRANCH, CHANNEL, COMMIT_HASH or FULLVERSION!"
```

Comment:
Is the `CHANNEL` still need to be here ?

Reply:
Yes, it's needed to check the result of `algod -v`:
```
if [[ "$STR" =~ .*"$FULLVERSION.$CHANNEL [$BRANCH] (commit #$SHORT_HASH)".* ]]
then
    echo -e "[$0] The result of \`algod -v\` is a correct match.\n$STR"
    exit 0
fi
```

Reply:
if that's the case, why you're not testing for it's existence on the `if` line ?

Reply:
I see what you mean.  I had a default value set before that I since removed.  I'll add the `CHANNEL` to the `if` conditional.